.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thanks for your contribution! Take a moment to answer these questions so that reviewers have the information they need to properly understand your changes:
+
+* What is the current state of things and why does it need to change?
+* What is the solution your changes offer and how does it work?
+
+Are there any issues or other links reviewers should consult to understand this pull request better? For instance:
+
+* Fixes #12345
+* See: #67890
+-->
+
+## Examples
+
+<!--
+Are there any examples of this change being used in another repository?
+
+When considering changes to the MetaMask module template, it's strongly preferred that the change be experimented with in another repository first. This gives reviewers a better sense of how the change works, making it less likely the change will need to be reverted or adjusted later.
+-->
```

Comment:
This section is part of the module template. We can remove it in repos that derive from the module template.

``` suggestion
```

Reply:
The file is removed